### ISSUE TITLE
Override icon class names

### ISSUE THREAD

**Issue by vanillajonathan** [Ionaru/easy-markdown-editor#448]
**Is your feature request related to a problem? Please describe.**
I already use a icon set, and don't want to also use Font Awesome.

**Describe the solution you'd like**
A `iconClassNames` option which provides a dictionary/map to define the icon class names.

**Additional context**
See [Bootstrap Icons](https://icons.getbootstrap.com/).
- Bold - https://icons.getbootstrap.com/icons/type-bold/
- Italic - https://icons.getbootstrap.com/icons/type-italic/
- Strike through - https://icons.getbootstrap.com/icons/type-strikethrough/
- Header 1 - https://icons.getbootstrap.com/icons/type-h1/
- Header 2 - https://icons.getbootstrap.com/icons/type-h2/
- Header 3 - https://icons.getbootstrap.com/icons/type-h3/
- Code - https://icons.getbootstrap.com/icons/code/
- Quote - https://icons.getbootstrap.com/icons/quote/
- Ordered list - https://icons.getbootstrap.com/icons/list-ol/
- Unordered list - https://icons.getbootstrap.com/icons/list-ul/
- Link - https://icons.getbootstrap.com/icons/link/
- Image - https://icons.getbootstrap.com/icons/image/
- Table - https://icons.getbootstrap.com/icons/table/
- Horizontal rule - https://icons.getbootstrap.com/icons/hr/
- Preview - https://icons.getbootstrap.com/icons/eye/
- Side-by-side - https://icons.getbootstrap.com/icons/layout-split/
- Fullscreen - https://icons.getbootstrap.com/icons/fullscreen/
- Guide - https://icons.getbootstrap.com/icons/question-circle/
- Undo - https://icons.getbootstrap.com/icons/arrow-counterclockwise/
- Redo - https://icons.getbootstrap.com/icons/arrow-clockwise/

Would be:
```javascript
iconClassNames: {
    toggleBold: 'bi bi-type-bold',
    toggleItalic: 'bi bi-type-italic',
    toggleStrikethrough: 'bi bi-type-strikethrough',
    toggleHeading1: 'bi bi-type-h1',
    toggleHeading2: 'bi bi-type-h2',
    toggleHeading1: 'bi bi-type-h3',
    toggleCodeBlock: 'bi bi-code',
    toggleBlockquote: 'bi bi-quote',
    toggleOrderedList: 'bi bi-list-ol',
    toggleUnorderedList: 'bi bi-list-ul',
    drawLink: 'bi bi-link',
    drawImage: 'bi bi-image',
    drawTable: 'bi bi-table',
    drawHorizontalRule: 'bi bi-hr',
    togglePreview: 'bi bi-eye',
    toggleSideBySide: 'bi bi-layout-split',
    toggleFullScreen: 'bi bi-fullscreen',
    guide: 'bi bi-question-circle',
    undo: 'bi bi-arrow-counterclockwise',
    redo: 'bi bi-arrow-clockwise'
}
```

**Comment by vanillajonathan** [comment]
Starting out by refactoring into a map #454

**Comment by vanillajonathan** [comment]
Yes, but then integrator would have to manually recreate their own custom buttons for every button.

This commit separates concerns so it relies on a map which we can later override to make it easier for integrators to use their own icons.

**Comment by Ionaru** [comment]
While I appreciate the work you're doing to make the editor as user-friendly as possible, I fear that all this work will go to waste when #447 happens, especially for extra configuration options to handle things that are already possible using other methods.

**Comment by vanillajonathan** [comment]
Yeah, maybe, but then there is always the option for integrators to stay with the latest EasyMDE 2 version.

I would hope to see this merged.
